packaging/greenboot/functions.sh

```diff
@@ -15,19 +15,38 @@ OCGET_CMD="oc get ${OCCONFIG_OPT}"
 #
 # The base value for the timeout and the maximum boot attempts can be defined in
 # the /etc/greenboot/greenboot.conf file using the MICROSHIFT_WAIT_TIMEOUT_SEC
-# and GREENBOOT_MAX_BOOTS settings.
+# and GREENBOOT_MAX_BOOTS settings. These values can be in the [60..9999] range
+# for MICROSHIFT_WAIT_TIMEOUT_SEC and the [1..9] range for GREENBOOT_MAX_BOOTS.
 #
 # args: None
 # return: Print the recommended timeout value to stdout
 function get_wait_timeout() {
     # Source Greenboot configuration file if it exists
     local conf_file=/etc/greenboot/greenboot.conf
     [ -f "${conf_file}" ] && source ${conf_file}
+
+    # Read and verify the wait timeout value, allowing for the [60..9999] range
     local base_timeout=${MICROSHIFT_WAIT_TIMEOUT_SEC:-300}
+    local reSecs='^[1-9]{1}[0-9]{0,3}$'
+    if [[ ! ${base_timeout} =~ $reSecs ]] ; then
+        base_timeout=300
+        echo "MICROSHIFT_WAIT_TIMEOUT_SEC value '${MICROSHIFT_WAIT_TIMEOUT_SEC}' is invalid: using '${base_timeout}' instead"
```

Comment:
It would be more helpful to say something like "could not parse" rather than just "invalid" here, so an admin who sees the message understands what is wrong.